### PR TITLE
Adding EMA Implementation

### PR DESCRIPTION
**Pull request by jayasreemca**
Adding EMA Implementation

### CODE REVIEW

File: it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+class IndividualsChargesAuthMainAgentsOnlyISpec extends AuthMainAgentsOnlyISpec {
+
+  val callingApiVersion = "3.0"
+
+  val supportingAgentsNotAllowedEndpoint = "create-amend-pension-charges"
+
+  private val taxYear    = TaxYear.fromMtd("2023-24")
+
+  val mtdUrl = (s"/pensions/$nino/${taxYear.asMtd}")
```
Comment (sudoku007) [inline it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala:34]: Can you remove the parenthesis as its not needed.

File: it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  def sendMtdRequest(request: WSRequest): WSResponse = await(request.put(fullValidJson))
+
+  val downstreamUri: String = s"/income-tax/charges/pensions/${taxYear.asTysDownstream}/$nino"
+    //s"/income-tax/${taxYear.asTysDownstream}/income-sources/charges/pensions/23-24/$nino"
```
Comment (sudoku007) [inline it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala:39]: Can you remove this comment line

File: it/auth/IndividualsChargesAuthSupportingAgentsAllowedISpec.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import v2.createAmend.def1.fixture.CreateAmendPensionChargesFixture.fullValidJson
+
+class IndividualsChargesAuthSupportingAgentsAllowedISpec extends AuthSupportingAgentsAllowedISpec {
+
+  val callingApiVersion = "3.0"
+
+  val supportingAgentsAllowedEndpoint = "create-amend-pension-charges"
+
+  private val taxYear    = TaxYear.fromMtd("2023-24")
+  val mtdUrl = (s"/pensions/$nino/${taxYear.asMtd}")
```
Comment (sudoku007) [inline it/auth/IndividualsChargesAuthSupportingAgentsAllowedISpec.scala:33]: Can you remove the parenthesis as its not needed.

File: it/auth/IndividualsChargesAuthSupportingAgentsAllowedISpec.scala [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+  def sendMtdRequest(request: WSRequest): WSResponse = await(request.put(fullValidJson))
+
+  val downstreamUri: String = s"/income-tax/charges/pensions/${taxYear.asTysDownstream}/$nino"
+    //s"/income-tax/${taxYear.asTysDownstream}/income-sources/charges/pensions/23-24/$nino"
```
Comment (sudoku007) [inline it/auth/IndividualsChargesAuthSupportingAgentsAllowedISpec.scala:39]: Can you remove the comment.

**Review by sudoku007 — CHANGES_REQUESTED** [review]

File: it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala (hunk 1 above)
Comment (jayasreemca) [reply it/auth/IndividualsChargesAuthMainAgentsOnlyISpec.scala]: Done and pushed.

**Review by sudoku007 — APPROVED** [review]

**Pull request merged**